interlok-json-streaming/src/main/java/com/adaptris/core/json/streaming/JsonPathStreamingService.java

```diff
@@ -0,0 +1,267 @@
+package com.adaptris.core.json.streaming;
+
+import com.adaptris.annotation.AdapterComponent;
+import com.adaptris.annotation.AutoPopulated;
+import com.adaptris.annotation.ComponentProfile;
+import com.adaptris.annotation.DisplayOrder;
+import com.adaptris.core.AdaptrisMessage;
+import com.adaptris.core.ServiceException;
+import com.adaptris.core.common.Execution;
+import com.adaptris.core.common.PayloadStreamInputParameter;
+import com.adaptris.core.services.path.json.JsonPathServiceImpl;
+import com.adaptris.core.util.ExceptionHelper;
+import com.adaptris.interlok.config.DataInputParameter;
+import com.adaptris.interlok.config.DataOutputParameter;
+import com.thoughtworks.xstream.annotations.XStreamAlias;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.Setter;
+import org.apache.commons.lang.ObjectUtils;
```

Comment:
lang3 -> and then you don't have to cast in the surfer() method.
It kinda sucks that commons-lang2 is still pulled in from downstream dependencies